### DISCUSSION TITLE
Error code PASSWORD_TOO_WEAK

### DISCUSSION THREAD

**Discussion by iBobik** [contember/admin#653] [Support]
Hello,
when resetting password, there is possible error code PASSWORD_TOO_WEAK.

[I was looking for](https://github.com/search?q=repo%3Acontember%2Finterface%20PASSWORD_TOO_WEAK&type=code) actual rules for password strength, so I have to tell the user, but have not found them. Does it mean there are not rules implemented?

(Btw, I does not need to have this validated, just clarify it and possibly cancel the error code.)

**Comment by jonasnobile** [comment]
Hi, password check with rules is implemented in Engine (https://github.com/contember/engine/blob/master/packages/engine-tenant-api/src/model/utils/password.ts). There is only one rule - the length of password has to be more than 6 characters.
